autogen/agentchat/contrib/orchestrator.py

```diff
@@ -62,6 +62,17 @@ def _broadcast(self, message, out_loud=[], exclude=[]):
             else:
                 self.send(message, a, request_reply=False, silent=True)
 
+    def _create_with_images(self, messages, **kwargs):
```

Comment:
question: when will we hit the 10 image limitation? Seems there's only 1 image_url in the llm request